libraries/lib-effects/EffectOutputTracks.h

```diff
@@ -29,8 +29,9 @@ class EFFECTS_API EffectOutputTracks
    static int nEffectsDone;
    static void IncEffectCounter() { ++nEffectsDone; }
 
+   using Interval = std::pair<double, double>;
```

Comment:
I've been using `TimeInterval` on my other branch, which I think is appropriate since it's a pair of doubles.

Reply:
I will change the name